ui/src/routes/(platform)/cluster/pods/pod.test.ts

```diff
@@ -1,18 +1,18 @@
 // +page.test.js
-import {describe, expect, test} from 'vitest';
-import {render, waitFor} from '@testing-library/svelte';
+import { describe, expect, test } from 'vitest';
+import { render, waitFor } from '@testing-library/svelte';
 import Page from './+page.svelte';
-import {FakePodRepository} from "../../../../../tests/fakerepo";
+import { FakePodRepository } from '../../../../../tests/fakerepo';
 
 describe('pod page', () => {
-    test('pod rendering', async () => {
-        const {getByText} = render(Page, {
-            data: {
-                repo: new FakePodRepository()
-            }
-        });
-
-        await waitFor(() => expect(getByText('pod1')).toBeDefined());
-        expect(getByText('pod2')).toBeDefined();
+  test('pod rendering', async () => {
+    const { getByText } = render(Page, {
+      data: {
+        repo: new FakePodRepository()
+      }
     });
+
+    await waitFor(() => expect(getByText('pod1')).toBeDefined());
```

Comment:
Does using `waitFor` and `getByText` here work any better than using `await` and `findByText`?

i.e.
```JavaScript
expect(await findByText('pod1')).toBeDefined()
```

Reply:
Good call, will update to `await screen.findByText('pod1');`